lib/perl/Genome/Model/ClinSeq/Command/Converge/SnvIndelReport.pm

```diff
@@ -857,7 +860,8 @@ sub parse_read_counts{
       my $prefix = $samples{$sample_name}{prefix};
       my $coverage = $samples{$sample_name}{coverage};
       #don't apply min_coverage on rnaseq, the transcript might not be expressed.
-      $min_coverage_observed = $coverage if ($coverage < $min_coverage_observed and $prefix !~ /rnaseq/);
+      $min_coverage_observed = $coverage if ($coverage < $min_coverage_observed and
```

Comment:
Some people would prefer that complex conditionals not be used in postfix notation.